src/explorer/SiteTreeItem.ts

```diff
@@ -167,6 +180,34 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.showBuildDuringDeployPrompt, false);
         }
     }
+
+    private async promptToSaveDeployConfigs(node: IAzureNode<SiteTreeItem>, currentWorkspacePath: string, fsPath: string): Promise<void> {
```

Comment:
What's the difference between 'currentWorkspacePath' and 'fsPath'? Please change the names to be more descriptive